library/fsharp

```diff
@@ -1,5 +1,14 @@
-# maintainer: Henrik Feldt <henrik@haf.se> (@haf)
+Maintainers: Henrik Feldt <henrik@haf.se> (@haf)
+GitRepo: https://github.com/fsprojects/docker-fsharp.git
 
-4.0.0.4: git://github.com/fsprojects/docker-fsharp@57b3ddadc85a63bf586ab89b523bb8326dad9c12 4.0.0.4
-4.0.1.1: git://github.com/fsprojects/docker-fsharp@57b3ddadc85a63bf586ab89b523bb8326dad9c12 4.0.1.1
-latest: git://github.com/fsprojects/docker-fsharp@57b3ddadc85a63bf586ab89b523bb8326dad9c12 4.0.1.1
+Tags: latest, 4, 4.1, 4.1.0.1
+GitCommit: 289a5e0066c69255c055dd5ee22f2e583d37fab4
+Directory: 4.1.0.1
+
+Tags: 4.0, 4.0.1, 4.0.1.1
+GitCommit: 289a5e0066c69255c055dd5ee22f2e583d37fab4
+Directory: 4.0.1.1
+
+Tags: 4.0.0.4
```

Comment:
I'd like to leave it in for a while longer, if that's OK with you. We actually still support F# 3.1 as well; things are stable over here, but there's no need for a docker container for it, because the next major versions are ABI-compatible.